core/sys/darwin/Foundation/NSString.odin

```diff
@@ -54,15 +51,6 @@ foreign Foundation {
 	StringFromClass :: proc(cls: Class) -> ^String ---
 }
 
-AT :: MakeConstantString
```

Comment:
This should be kept in, even though it is technically CoreFoundation, it works and is used in a lot of examples and other applications today